src/header.rs

```diff
@@ -181,7 +181,7 @@ impl Header {
         header
     }
 
-    fn is_ustar(&self) -> bool {
+    pub(crate) fn is_ustar(&self) -> bool {
```

Comment:
Instead of making this `pub(crate)`, I think the same can be achieved with `header.as_ustar().is_some()`